test/utils/OWNERS

```diff
@@ -0,0 +1,44 @@
+reviewers:
```

Comment:
Created `test/integration/deployment/OWNERS` instead. Agree on the messiness -- we should clean it up in follow up PRs.